packages/env/lib/test/execute-lifecycle-script.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* eslint-disable jest/no-conditional-expect */
+/**
+ * Internal dependencies
+ */
+const {
+	LifecycleScriptError,
+	executeLifecycleScript,
+} = require( '../execute-lifecycle-script' );
+
+describe( 'executeLifecycleScript', () => {
```

Comment:
I don't know if this is critical enough to change immediately, but I don't think this test suite would work cross platform (like if someone is working on unit tests in Windows). I think the core repos typically try to work cross platform as much as possible

Reply:
I changed the test to make it more platform-agnostic. I used `node` commands instead because it's virtually impossible for that to not work.